codalab/server/auth.py

```diff
@@ -66,4 +66,6 @@ def _make_token_request(self, data):
         except urllib.error.HTTPError as e:
             if e.code == 401:
                 return None
-            raise
+            raise e
+        except urllib.error.URLError:
+            print("Your login info is incorrect. Please double check")
```

Comment:
In your latest screenshot, it looks like both error messages show up:

"Your login info is incorrect. Please double check"
'Login credentials are incorrect. Please double check'

Reply:
Yeah, that's because they come from two separate processes. I can definitely remove one of them, but then we don't know where exactly are errors thrown. How do you think?

Reply:
We should only show one of these errors.